startup.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
I would have created this commit in a separate pull request, because it doesn't feel related to this story.
No need to change it now, but might be worth considering for incoming PRs